src/hotspot/share/oops/method.hpp

```diff
@@ -134,7 +134,8 @@ class Method : public Metadata {
 
   virtual bool is_method() const { return true; }
 
-  void restore_unshareable_info(TRAPS);
+  void remove_unshareable_info()        NOT_CDS_RETURN;
+  void restore_unshareable_info(TRAPS)  NOT_CDS_RETURN;
```

Comment:
Surely these should ifdef'd out not stubbed out? The callers of these should also be in INCLUDE_CDS blocks.

Reply:
Fixed.